examples/wifinina/http-get/main.go

```diff
@@ -27,23 +27,21 @@ import (
 // access point info
 const ssid = ""
 const pass = ""
+const timeout = time.Minute
 
-// IP address of the server aka "hub". Replace with your own info.
 // Can specify a URL starting with http or https
-const url = "http://tinygo.org/"
+const url = "https://raw.githubusercontent.com/tinygo-org/tinygo/release/LICENSE"
```

Comment:
I think this change is very good.
I would prefer https, but I know there is a #290 problem with https.

Reply:
It works on my board, Nano RP2040 Connect. But then, I've updated it to latest nina-fw (1.4.8)

Reply:
I had only tried it with 1.4.5. I'll try it later. Thanks for the information.